/**/controller_manager:
  ros__parameters:
    update_rate: 1000  # Hz

    # joint_state_controller:
    #   type: joint_state_controller/JointStateController

    joint_group_position_controller:
      type: position_controllers/JointGroupPositionController
      
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    attitude_controller:
      type: flight_controllers/SimulationAttitudeController

# /**/joint_state_controller:
/**/joint_group_position_controller:
  ros__parameters:
    joints:
      - joint1
      - joint2
      - joint3
    interface_name: position
    gains:
        p: 100.0
        i: 0.0
        d: 10.0
    state_publish_rate: 25

/**/attitude_controller:
  ros__parameters:
    imu: "spinal_imu"
    mag: "spinal_mag"

/**/sim_param_server:
  ros__parameters:

    ground_truth_pub_rate: 0.01
    ground_truth_pos_noise: 0.001
    ground_truth_vel_noise: 0.005
    ground_truth_rot_noise: 0.001
    ground_truth_angular_noise: 0.005

    ground_truth_vel_drift: 0.1
    ground_truth_rot_drift: 0.1
    ground_truth_angular_drift: 0.01

    ground_truth_vel_drift_frequency: 1.0
    ground_truth_rot_drift_frequency: 1.0
    ground_truth_angular_drift_frequency: 0.01

    mocap_pub_rate: 0.01
    mocap_pos_noise: 0.001
    mocap_rot_noise: 0.001

